aw_server/custom_watcher_pages.py

```diff
@@ -0,0 +1,44 @@
+import logging
```

Comment:
Would be nice with a docstring describing basic usage/intent :)

Reply:
Of course, sorry I forgot about that.